src/coprocessor/codec/chunk/chunk.rs

```diff
@@ -112,20 +121,25 @@ impl<'a> Row<'a> {
     }
 
     /// Get the row index of Chunk.
+    #[inline]
     pub fn idx(&self) -> usize {
         self.idx
     }
 
     /// Get the number of values in the row.
+    #[inline]
     pub fn len(&self) -> usize {
         self.c.num_cols()
     }
 
+    /// Get the datum of column with specified type in the row.
```

Comment:
"Get the datum of **a** column with **the** specified type in the row."
OR 
"Get the datum of **the** column with **the** specified type in the row."?